### ISSUE TITLE
Disable or reduce snap delay

### ISSUE THREAD

**Issue by mval1002** [bfwg/ngx-drag-scroll#159]
* **I'm submitting a ...**
  - [ ] bug report
  - [x] feature request
  - [ ] support request => Please do not submit support request here, see note at the top of this template.

* **Do you want to request a *feature* or report a *bug*?**
   - Request a feature/ determine if feature is already present. 

* **What is the current behavior?**
   - Snap scroll has a short, but obvious delay before continuing to its final destination.

* **What is the expected behavior?**
   - Not so much expected, but I would like to eliminate or reduce the delay seen when snap is on. After scrolling the scroll comes to a complete stop for a noticeable amount of time, and I think it would feel snappier if the scroll continued.

* **What is the motivation / use case for changing the behavior?**
   - Smoother transitioning from one element to another

* **Please tell us about your environment:**
   - Version: 2.0.0-beta3
   - Browser: [ Chrome (v67) ]

Thanks for this great component! I would like to make very clear that this is far from a big issue. If this is already possible, but not documented that would be great, but otherwise I saw that you've been rather busy lately. Regardless, this component does an amazing job!

**Comment by bfwg** [comment]
Hi @mval1002 , in 2.0.0-beta-4, we have introduced an new input `snap-duration`. From there, you should be able to control the snap animation duration.

**Comment by mval1002** [comment]
Hi @bfwg! Thank you for the update, and sorry for taking some time to respond. I got caught up in some other aspects of my project and just now got a chance to try this!

I updated to Beta4 and added a `snap-duration="10"` input to my `drag-scroll` element, but it doesn't seem to make a difference. Am I missing a step? 

Also, based on the description in the docs, I fear I may not have given a good enough explanation of my request. The animation itself is fine for my purposes, but BEFORE the animation starts, the scroll window just sits for a short time. 

- Mouse down
   - Move
- Mouse up
- Nothing 
   - This is what I'd like to reduce/remove, its probably around 250-500ms
- Snap Animation starts
   - snap-duration (?)
- Snap Animation end

Again this is not at all a deal breaker, the fact that there is a snap feature at all is actually amazing! It's something that was there before I even knew I needed it, hahaha! Even if the snap-duration doesn't affect the thing I brought up, I'm sure it will be very useful for many people! I really do appreciate your effort for this awesome component!
